lib/team_result_parser.rb

```diff
@@ -96,28 +96,18 @@ def get_win_percentage(season_games)
         end
       end
     end
-    # Add default value for hash (Hash.new(0))
-    coach_played = {}
+    coach_played = Hash.new(0)
```

Comment:
Great idea on cutting some lines by calling `Hash.new` here and defaulting it's value to `0`